spec/plugins/hmac-auth/access_spec.lua

```diff
@@ -0,0 +1,150 @@
+local spec_helper = require "spec.spec_helpers"
+local http_client = require "kong.tools.http_client"
+local cjson = require "cjson"
+local sha1 = require "sha1"
+local base64 = require "base64"
+
+local json_decode = cjson.decode
+local os_date = os.date
+local base64_encode = base64.encode 
```

Comment:
It is not useful to perform global optimisations on tests, it only makes them less readable.